src/shared/helpers.h

```diff
@@ -108,6 +109,13 @@ RC_UNUSED static void xclose_memstream(FILE *f)
 
 #undef ERRX
 
+RC_UNUSED static size_t clock_ms(void)
```

Comment:
Use `int64_t`. We don't want to be stuck with 32 bit time on 32 bit system. Also, it's important to use signed so that subtraction work naturally.